aws-case-study.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Amazon Web Services Case Study
+keywords: development platforms, AWS, Cloud Hosting
+tags: Development Platforms, AWS, Cloud Hosting
+sidebar: development-platforms
+permalink: development-platforms/example-product.html
+folder: development-platforms
+---
+
+## Introduction
+
+Amazon Web Services was the first major cloud hosting service. Cloud Hosting is a service that uses 
```

Comment:
There is no need to break lines artificially. Text is wrapped properly when it is rendered. Please remove all of these line breaks mid sentence.

Reply:
The text wrap is fixed. Thank you